### PR TITLE
[Doc] Updated documentation and one tiny code change.

### PR DESCRIPTION
**Pull request by paul-amonson**
Signed-off-by: Paul Amonson <paul.d.amonson@intel.com>

### CODE REVIEW

File: docs/UnderstandingDaiNetworkListenerConfigurations.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ If the _logProvider_ key is missing then *log4j2* is assumed.
 
 If the _useBenchmarkingActions_ key is missing __false__ is assumed for normal operations of system actions like
  storing and publishing data. If __true__ then the normal actions are replaced with metrics gathering and more log
-  output to _Logger.info()_ are made for gathering timing data. 
+  output to _Logger.info()_ are made for gathering timing data. You may also use the environmental variable _DAI_USE_BENCHMARKING_ wit hteh same __true__ or __false__ values.
```
Comment (nbwhite) [inline docs/UnderstandingDaiNetworkListenerConfigurations.md]: Typo, "with the same"

File: docs/UnderstandingDaiNetworkListenerConfigurations.md [hunk 2]
```diff
@@ -25,13 +25,13 @@ If the _logProvider_ key is missing then *log4j2* is assumed.
...
 
 ## 3. Subjects Map (`subjectMap`)
-This lists all possible subjects (or topics) supported by the *PartitionedMonitorAdapter*. The names are mapped to the *DataType* enum values internal to the adapter. This table should not be changed unless you also change the source code as well.
+This lists all possible subjects (or topics) supported by the *PartitionedMonitorAdapter*. The names are mapped to the *DataType* enum values internal to the adapter. This table should not be changed unless you also change the source code as well. This map should never change and in the future will be moved to a Java resource.
```
Comment (nbwhite) [inline docs/UnderstandingDaiNetworkListenerConfigurations.md]: Missed reference to PartitionedMonitorAdapter

File: docs/UnderstandingDaiNetworkListenerConfigurations.md [hunk 3]
```diff
@@ -180,13 +160,12 @@ The name field of this type is always `rabbitmq`.
 ...
 ```
 ## 7 Adapter Profiles (adapterProfiles)
-This section defines all of the possible profiles that can be used by the *PartitionedMonitorAdapter* class.  Each profile describes a single data transformer provider, action provider, subject(s) provided, and the network stream(s) used to gather the data. Example:
+This section defines all of the possible profiles that can be used by the specific provider's implementation using the *dai_network_listener* component.  Each profile describes a single data and acion transformer provider, subject(s) provided, and the network stream(s) used to gather the data. Example:
```
Comment (nbwhite) [inline docs/UnderstandingDaiNetworkListenerConfigurations.md]: typo "action transformer"

File: docs/UnderstandingDaiNetworkListenerConfigurations.md [hunk 4]
```diff
@@ -198,18 +177,16 @@ This section defines all of the possible profiles that can be used by the *Parti
 }
 ...
 ```
-In this example the name of the single profile is `allTelemetry` which will be passed as the 4th parameter on the adapter’s application command line:
+In this example the name of the single profile is `allTelemetry` which will be listed as an *Parms* in the **MachineConfig.json** file in the section *InitialWorkItems*:
```
Comment (nbwhite) [inline docs/UnderstandingDaiNetworkListenerConfigurations.md]: typo, should be "a Parms"

Comment (paul-amonson) [reply docs/UnderstandingDaiNetworkListenerConfigurations.md]: I can't see the issue here? Parms was copy/pasted from the MachineConfig.json file.

Comment (paul-amonson) [reply docs/UnderstandingDaiNetworkListenerConfigurations.md]: Ok, I see it.

**Pull request merged**